src/promptflow-evals/tests/evals/conftest.py

```diff
@@ -115,6 +118,33 @@ def project_scope() -> dict:
     return dev_connections[conn_name]["value"]
 
 
+@pytest.fixture
+def mock_trace_destination_to_cloud(project_scope: dict):
+    """Mock trace destination to cloud."""
+
+    subscription_id = project_scope["subscription_id"]
+    resource_group_name = project_scope["resource_group_name"]
+    workspace_name = project_scope["project_name"]
+
+    trace_destination = (
+        f"azureml://subscriptions/{subscription_id}/resourceGroups/{resource_group_name}/"
+        f"providers/Microsoft.MachineLearningServices/workspaces/{workspace_name}"
+    )
+    with patch("promptflow._sdk._configuration.Configuration.get_trace_destination", return_value=trace_destination):
```

Comment:
For user, how would they set this trace destination?

Reply:
Had a offline chat this is temporary until a bug is fixed